app/views/entities/index.html.erb

```diff
@@ -39,7 +39,7 @@
       </div>
     <% end %>
   </div>
-  <div class="row mt-4 text-center justify-content-center">
-    <%= link_to "New Entity", new_group_entity_path(@group), class: "btn btn-primary col-md-4" %>
+  <div class="container text-center bottom-element">
+    <%= link_to "New Entity", new_group_entity_path(@group), class: "btn btn-primary w-100" %>
```

Comment:
- [ Required ] I think it would be nice if you implement the previous reviewer's suggestion in the exact manner he described, the `New Entity` button should have the `greenish` color and `whitish` border as illustrated on the Behance design. Currently, your design has a `bluish` button without a `whitish` border around it. Please see the screenshot below and rectify this.

## App Screenshot vs Design screenshot
| Your design | Behance design |
|-----------------|------------------------|
| ![Screenshot from 2023-11-26 02-16-02](https://github.com/chudisoft/budget_manager/assets/16392046/ac6592f9-68f8-47ab-895b-ef94abef6090) | ![Screenshot from 2023-11-26 02-12-48](https://github.com/chudisoft/budget_manager/assets/16392046/8e09139d-bf4b-4cb9-86a0-8c4e8a1ea599) |